src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements AreaCalculetion {

```

Comment:
The interface name `AreaCalculetion` is not following Java naming conventions. It should be a noun and also spelled correctly, such as `AreaCalculation`.